average.js

```diff
@@ -1,6 +1,10 @@
-
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  numbers = numbers.filter(function(val) {
+    val = parseFloat(val).toFixed(2);
+    return !isNaN(val) && val <= 100 && val >= -100 ;
```

Comment:
The code removes outliers, but the test exercises decimal points?
How did you check that the code works fine?